compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -2285,8 +2285,8 @@ impl<'tcx> Debug for Rvalue<'tcx> {
 /// Constants
 ///
 /// Two constants are equal if they are the same constant. Note that
-/// this does not necessarily mean that they are "==" in Rust -- in
-/// particular one must be wary of `NaN`!
+/// this does not necessarily mean that they are `==` in Rust -- in
+/// particular, one must be wary of `NaN`!
```

Comment:
Would this be better?
```suggestion
/// this does not necessarily mean that they are `==` in Rust. In
/// particular, one must be wary of `NaN`!
```